tests/main/interfaces-snapd-control-with-manage/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Ensure that the snapd-control "refresh-schedule" attribute works.
+
+prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local test-snapd-control-consumer
+    
+    # now pretend test-snapd-control-consumer comes from the store (fugly HACK)
```

Comment:
we could use the fakestore here instead (it could help also with setting up assertions), it's a bit of work though